### DISCUSSION TITLE
Vote for your favorite next big feature  🙌

### DISCUSSION THREAD

**Discussion by SkalskiP** [SkalskiP/make-sense#269] [Polls]
Hi there 👋!

We are grateful that our community, has no shortage of ideas on how to make Make Sense even better. There's nothing greater than seeing how involved you are in the development of the tool. We sincerely thank you for that. 🙏 

There are currently several dozen ideas on our TODO list. We love them all and they all deserve to eventually be included in Make Sense. Unfortunately, our, time is limited and we often have to make difficult decisions, such as those regarding the prioritization of features.

In the past, the dev team independently decided the order of task execution. However, we decided that it would be a great idea to ask you for your opinion. After all, no one knows better than you what you really need. Please take a peek at the list below and cast your vote for the feature you think is most valuable. We can't wait for your response. 😎

- Label statistics to better understand your dataset #76
- Image tweaking to make labeling easier - brightness, gamma, contrast, and grayscale #207
- Loading custom YOLOv5 model to speed up labeling #156
- Allow for saving and loading your project from hard drive #101

**Comment by TomerGamerTV** [comment]
any updates? it's almost 2024 already

**Comment by jsomeara** [comment]
any updates? it's almost 2025 already

**Reply by SkalskiP** [reply]
hi @jsomeara 👋🏻 make-sense is no longer actively developed :/

**Reply by dieperdev** [reply]
@SkalskiP you may want to update the readme so others are aware
